tool/tas.re

```diff
@@ -171,6 +179,9 @@ static mut TAS_COMPONENT = Component {
             TAS_STATE.is_recording = !TAS_STATE.is_recording;
             if TAS_STATE.is_recording {
                 TAS_STATE.recording = List::new();
+                TAS_STATE.recording_start_timestamp = current_time_millis();
+            } else {
+                TAS_STATE.recording_end_timestamp = current_time_millis();
```

Comment:
Can you save a recording before having ended it? If yes, that way it might be possible to have an end-time of `0`.

Reply:
You can. To prevent this, I could maybe add a check like I do for the filename, that says something like `ERROR: cannot save file whilst recording is active`